classes/issue/Issue.inc.php

```diff
@@ -536,6 +536,20 @@ function hasDescription() {
 	}
 
 	/**
+	 * Check whether this issue is a current issue
+	 * @return bool
+	 */
+	function isCurrent() {
```

Comment:
I chose to declare another function because the getCurrent function returns the value of the DB. The isCurrent could incorporate more logic. But I could erase that, and keep the getCurrent until the "more logic" is needed....